ui/app/AppLayouts/Profile/views/ChangePasswordView.qml

```diff
@@ -12,27 +12,82 @@ import StatusQ.Controls 0.1
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
 
 import AppLayouts.Profile.popups 1.0
 
 SettingsContentBase {
     id: root
 
     property var privacyStore
+
+    readonly property bool biometricsEnabled: localAccountSettings.storeToKeychainValue === Constants.keychain.storedValue.store
+    onBiometricsEnabledChanged: {
+        biometricsSwitch.checked = Qt.binding(() => { return root.biometricsEnabled });
```

Comment:
The `biometricsSwitch` is in a loader. On the first event the `biometricsSwitch` may not exist. An undefined check would be useful here.

Reply:
there is a warning when this component gets loaded that the `biometricsSwitch` is undefined. We could remove it this way.

Reply:
@alexjba I don't see any warning but I've added the check anyways. in regards to the buttons cut, I can't see that one either